zipline-testing/build.gradle.kts

```diff
@@ -68,7 +68,6 @@ kotlin {
 }
 
 tasks {
-  // TODO upstream this to ZiplinePlugin
   withType<KotlinJsCompile>().all {
     kotlinOptions {
       freeCompilerArgs += listOf("-Xir-per-module")
```

Comment:
Delete it all. This is the default!

Reply:
https://github.com/cashapp/zipline/pull/1179